### PR TITLE
[Graph Browser 2] add webdriver tests

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: server/routes/browser.py [hunk 1]
```diff
@@ -29,7 +29,8 @@ def browser_main():
 @bp.route('/<path:dcid>')
 def browser_node(dcid):
     if os.environ.get('FLASK_ENV') == 'svobs' or os.environ.get(
```
Comment (beets) [inline server/routes/browser.py:31]: you could also do
```
if os.environ.get('FLAS_ENV') in ['svobs', 'local-svobs', 'webdriver']
```

**Review by beets — APPROVED** [review]
Thanks for adding these tests!

We should also add tests for each PageDisplayType. For nybg, we can test that images are able to load.
There should be a basic test for the stat var hierarchy too (maybe exercise a simple search path)
For the charts, maybe a test for clicking on the point? It's possible to search for svg nodes via xpath.

File: server/routes/browser.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/browser.py]: done

File: server/webdriver_tests/browser_test.py [hunk 2]
```diff
@@ -186,8 +188,107 @@ def test_page_serve_ca_population(self):
...
+        WebDriverWait(self.driver, self.TIMEOUT_SEC).until(element_present)
+        first_result = self.driver.find_element_by_xpath(
+            '//*[@id="stat-var-hierarchy-section"]/div/div[1]/div[2]/div/div[1]'
+        )
+        first_result.click()
+
+        # Assert that the section Count_Person_Male_AsianAlone opened and shows at least one chart
+        element_present = EC.presence_of_element_located((
+            By.XPATH,
+            '//*[@id="stat-var-hierarchy-section"]/div/div[2]/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div[11]/div/div/div/div/div/div[1]/div[2]'
```
Comment (beets) [inline server/webdriver_tests/browser_test.py:275]: this is incredibly brittle... are there classes you could add to make this easier to update in the future (e.g. is-open)?

**Review by beets — COMMENTED** [review]
thanks for adding the additional tests!

File: server/webdriver_tests/browser_test.py (hunk 2 above)
Comment (chejennifer) [reply server/webdriver_tests/browser_test.py]: done

**Review by beets — APPROVED** [review]
thx!

File: server/webdriver_tests/browser_test.py (hunk 2 above)
Comment (beets) [reply server/webdriver_tests/browser_test.py]: i wonder if it could be simplified more, but this is much better

**Pull request merged**